docker/local_batch_job/README.md

```diff
@@ -1,47 +1,21 @@
-# Run openEO from Docker file (experimental)
+# Run openEO Geotrellis locally using Docker
 
-This tool allows to run simple openEO processes. External collections can be loaded with load_stac.
-prerequisites: Linux with Docker already installed.
+This tool allows to run simple openEO processes. Collections can be loaded with load_stac.
 
-- clone this repository, or download just the `local_batch_job` folder
-- Build the image first with `sudo docker build -t openeo_docker_local .`
-- Run graph with `./local_batch_job.sh path/to/process_graph.json`
-- The output files will be written to the same folder as process_graph.json
+Note that the openEO Geotrellis backend was designed primarily to run on a distributed processing cluster.
+The purpose of this tool is for instance to allow faster iterations on small test datasets, without depending on an online deployment.
+It should also work for larger workloads, but this may require sufficient IT skills or insight into the workings of a Spark application.
+For users that require support, we recommend using one of the online deployments.
+
+Prerequisites: Docker already installed. Python with the openEO client installed.
+Note, you can refer to local stac collections by file path on Linux, but not on Windows.
 
-## Use local STAC catalogs
-The easiest is to host the local stac catlog in a local web-server. Giving fast access from within the Docker container.
+- Clone / download this repository
+- Build the image with `cd openeo-geopyspark-driver && sudo docker build -t openeo_docker_local . -f docker/local_batch_job/Dockerfile`
+- Run graph with `./local_batch_job path/to/process_graph.json folder/that/contains/local/stac/catalogs/`
```

Comment:
what is `./local_batch_job`  ?

should this be  `run_graph_locally.py` as defined in setup.py?

Reply:
if so, then the usage instruction should also state that openeo-geopyspark-driver must be installed,  e.g. in a local virtual env

Reply:
`openeo-geopyspark-driver` is set-up inside the docker container. local_batch_job will just launch run_graph_locally.py inside the docker container correctly.
But the new entrypoint needs to go trough Jenkins before it is usable here. So a bit of chicken and egg.

Reply:
I merged already so the base docker image will be rebuild with the new entrypoint

Reply:
Will be fixed here: https://github.com/Open-EO/openeo-geopyspark-driver/issues/894